Promote PBS constants to optional parameters (#121)
CABLE-LSM/benchcab: docs/user_guide/config_options.md

: NCI modules to use for compiling CABLE

### `pbs`

> : Contains sub-keys which specify settings specific to the PBS scheduler at NCI.

Reviewer: ```suggestion
: Contains settings specific to the PBS scheduler at NCI for the PBS script running the CABLE simulations at FLUXNET sites and the bitwise comparison for these simulations.
```

#### `ncpus`

> : The number of CPU cores to allocate for the main job script, i.e. the `-l ncpus=<4>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]). 

Reviewer: ```suggestion
: The number of CPU cores to allocate for the PBS job, i.e. the `-l ncpus=<4>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]). 
```

: This key is **optional** and can be omitted from the config file. By default `ncpus` is set to `18`.

#### `mem`

> : The total memory limit for the main job script, i.e. the `-l mem=<10GB>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]).

Reviewer: ```suggestion
: The total memory limit for the PBS job, i.e. the `-l mem=<10GB>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]).
```

: This key is **optional** and can be omitted from the config file. By default `mem` is set to `30GB`.

#### `walltime`

> : The wall clock time limit for the main job script, i.e. `-l walltime=<HH:MM:SS>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]).

Reviewer: ```suggestion
: The wall clock time limit for the PBS job, i.e. `-l walltime=<HH:MM:SS>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]).
```

: This key is **optional** and can be omitted from the config file. By default `walltime` is set to `6:00:00`.

#### `storage`

> : A list of extra storage flags required for the main job script, i.e. `-l storage=<scratch/a00>` (see [PBS Directives Explained][nci-pbs-directives]).

Reviewer: ```suggestion
: A list of extra storage flags required for the PBS job, i.e. `-l storage=<scratch/a00>` (see [PBS Directives Explained][nci-pbs-directives]).
```

: NCI modules to use for compiling CABLE

### `fluxsite`

> : Contains settings specific to fluxsite tests.

Reviewer: ```suggestion
: Contains settings specific to fluxsite tests. 
: This key is **optional**. Default settings for the fluxsite tests will be used if it is not present
```

#### `pbs`

> : Contains settings specific to the PBS scheduler at NCI for the PBS script running the CABLE simulations at FLUXNET sites and the bitwise comparison for these simulations.

Reviewer: ```suggestion
: Contains settings specific to the PBS scheduler at NCI for the PBS script running the CABLE simulations at FLUXNET sites and the bitwise comparison for these simulations. 
: This key is **optional**. Default values for the PBS settings will apply if it is not specified.
```
